package.json

```
@@ -63,6 +63,8 @@
 		"vue": "^2.6.11",
 		"vue-echarts": "^4.0.4",
 		"vue-flatpickr-component": "^8.1.5",
+		"vue-i18n": "^8.15.3",
+		"vue-input-autowidth": "^1.0.9",
 		"vue-i18n": "^8.15.4",
```

seems like `vue-i18n` is now defined twice 🤔
```suggestion
		"vue-i18n": "^8.15.4",
		"vue-input-autowidth": "^1.0.9",
```